<launch>

  <!-- <arg name="recordData" default="false" /> -->

  <rosparam command="load" file="$(find hand_control)/param/control_blue.yaml" />

  <node pkg="openhand" name="openhandNode" type="openhandNode.py" output="screen">
		<rosparam command="load" file="$(find hand_control)/param/model_t42_blue.yaml"/>
  </node>

  <!-- <node name="move_gripper" pkg="move_gripper" type="move_gripper" output="screen"/>
  <node name="gripper_t42" pkg="gripper_nodes" type="gripper_node" output="screen">
    <rosparam command="load" file="$(find hand_control)/param/control_blue.yaml" />
  </node> -->

  <!-- <node name="gripper_state_publisher" pkg="gripper_state_publisher" type="gripper_state_publisher_node"/>

  <node pkg="hand_control" name="hand_control" type="hand.py" output="screen">
        <rosparam command="load" file="$(find hand_control)/param/control_blue.yaml" />
  </node>

  <include file="$(find marker_tracker)/launch/marker_tracker.launch"/> -->

</launch>
